package.json

```diff
@@ -10,7 +10,8 @@
     "build": "quasar build",
     "lint": "eslint --ext .js,.ts,.vue ./",
     "lint:fix": "eslint --fix --ext .js,.vue src",
-    "test": "jest --coverage"
+    "test": "jest --coverage",
+    "q": "jest test/jest/__tests__/antelope/stores/balances.spec.ts"
```

Comment:
can we remove `q`?

Reply:
also, forgot to remove it. It helps me to run my tests faster.